### DISCUSSION TITLE
can not access pages with recaptcha v3

### DISCUSSION THREAD

**Discussion by naols** [cefsharp/CefSharp#3520] [Q&A]
i have a functional algorithm about a month, after the update of chromium 4/21 stopped accessing some pages that have recaptcha v3 (before it worked perfectly) does it have something related to cef or always was so?

**Comment by amaitland** [comment]
> after the update of chromium 4/21

What update are you referring to exactly? Version number?

I'd suggest testing with the `CEF Sample Application` see https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef for details.

> does it have something related to cef or always was so?

`CEF` has it's own support forum at https://magpcss.org/ceforum/index.php where you can ask if anyone else is having problems.

**Reply by naols** [reply]
> 
> 
> > after the update of chromium 4/21
> 
> What update are you referring to exactly? Version number?
> 
> I'd suggest testing with the `CEF Sample Application` see https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef for details.
> 
> > does it have something related to cef or always was so?
> 
> `CEF` has it's own support forum at https://magpcss.org/ceforum/index.php where you can ask if anyone else is having problems.

I would like to make it clear that I am not sure that the update has raised my problem, however, note that the problem started to occur not occurring the same day of the update.
https://cef-builds.spotifycdn.com/index.html#windows64
04/21/2021 - 90.5.5 + gf718c89 + chrome-90.0.4430.72 / Chrome 90.0.4430.72

I tested using the client cef that passed me and I keep getting the same error (https://i.imgur.com/ZNX5NtA.png)

before I was able to access this link perfectly without errors, with your experience, do you believe that this is because of cef / cefsharp or the nike website that changed your score settings?

the cef browser always returns a low score, do you have any information on how to increase that score?

**Reply by amaitland** [reply]
> I tested using the client cef that passed me and I keep getting the same error (https://i.imgur.com/ZNX5NtA.png)

As the `CEF Sample application` appears to reproduce the problem I'd suggest asking on https://magpcss.org/ceforum/index.php

> before I was able to access this link perfectly without errors, with your experience, do you believe that this is because of cef / cefsharp or the nike website that changed your score settings?

If the problem occurs with the `CEF Sample Application` then this isn't a `CefSharp` specific problem. I really couldn't speculate as to what's changed. Maybe you can look at the `Recpatch` requests/responses in `DevTools` and see if you can get any clues.

> the cef browser always returns a low score, do you have any information on how to increase that score?

I'm not aware of any such information. Someone on ceforum might have some suggestions. 

My experience with `Recaptch` is that `Google` don't response to `Recaptch` based support questions very often.

**Reply by amaitland** [reply]
Looks like you already asked at https://magpcss.org/ceforum/viewtopic.php?f=6&t=18359

I don't have any suggestions currently.
